tests/rundb/test_httpdb.py

```diff
@@ -276,6 +277,34 @@ def test_artifacts(create_server):
     assert len(artifacts) == 0, "bad number of artifacts after del"
 
 
+def test_list_artifacts_filter_by_kind(create_server):
```

Comment:
Our plan is to move all of the tests in this file to be under `tests/integration/sdk_api`
So I prefer not to create ourselves more tests to migrate - can you please move this one there ?

Reply:
Moved this test and the other real artifacts test (there was another "test" there that did nothing - I removed it) from `httpdb` to a new file under `sdk_api`.